src/utils/filter_methods.ts

```diff
@@ -34,7 +33,7 @@ export function latestRevisionFilterForDrives(
 	_index: number,
 	allEntities: ArFSDriveEntity[]
 ): boolean {
-	const allRevisions = allEntities.filter((e) => e.driveId === entity.driveId);
+	const allRevisions = allEntities.filter((e) => e.driveId.equals(entity.driveId));
 	const latestRevision = allRevisions[0];
 	return entity.txId === latestRevision.txId;
```

Comment:
Shouldn't this be:

```suggestion
	return entity.txId.equals(latestRevision.txId);
```

Reply:
YES, and now the question is what tests aren't failing.

Reply:
The answer is that there are no tests since this (and perhaps others of the) filter(s) function(s) are used in read-only workflows that aren't setup for testability yet 😑 I'll add some and patch this.

Reply:
Actually it works because the filter functions are used in a way where you'll get referential equality on everything, but we should still test and update them.

Reply:
Updated in 1698875 thru a4a7dde